.env.example

```diff
@@ -0,0 +1,2 @@
+# This is only needed for real deployments to GCP
```

Comment:
Can we eliminate this option entirely for local development?  It's an odd valora production environment detail to leak into a builders dev workflow.

Reply:
yes removed.